frontend/src/pages/home/home.tsx

```diff
@@ -363,13 +371,14 @@ Thank you."}
                             </FadeIn>
                             <div className="project-separator" />
                             <FadeIn className="fade-in">
-                                <ProjectCard
+                                <MultiProjectCard
```

Comment:
Probably shouldn't be this PR, you could potentially consider using modal to zoom into the carousel like the ImageCard.